maestro/lifecycle.py

```diff
@@ -17,7 +17,7 @@
 class BaseLifecycleHelper:
     """Base class for lifecycle helpers."""
 
-    def test(self):
+    def test(self, ship=None, container=None):
```

Comment:
What is passed is the name of a container running on the ship, a <str>, not an 'Container' object.